src/libAtomVM/term.h

```diff
@@ -136,6 +136,11 @@ struct PrinterFun
     printer_function_t print;
 };
 
+typedef struct BinaryPosLen {
```

Comment:
Does this struct have any purpose outside `binary:match` or is it just a helper function for it?